src/aap_eda/settings/default.py

```diff
@@ -293,32 +312,74 @@ def _get_secret_key() -> str:
 
 RQ_UNIX_SOCKET_PATH = settings.get("MQ_UNIX_SOCKET_PATH", None)
 
-if RQ_UNIX_SOCKET_PATH:
-    RQ_QUEUES = {
-        "default": {
-            "UNIX_SOCKET_PATH": RQ_UNIX_SOCKET_PATH,
-            "DEFAULT_TIMEOUT": 300,
-        },
-        "activation": {
+# A list of queues to be used in multinode mode
+# If the list is empty, the single node mode is used
+WORKERS_RULEBOOK_QUEUES = settings.get("WORKERS_RULEBOOK_QUEUES", [])
```

Comment:
@Alex-Izquierdo RULEBOOK_WORKERS_QUEUES might be a better name

Reply:
changed.